docs/.vitepress/config.ts

```diff
@@ -265,7 +265,7 @@ export default defineConfig({
     ],
 
     socialLinks: [
-      { icon: 'github', link: 'https://github.com/mineiros-io/terramate' },
+      { icon: 'github', link: 'https://github.com/terramate-io/terramate' },
       { icon: 'discord', link: 'https://terramate.io/discord' },
       { icon: 'twitter', link: 'https://twitter.com/mineirosio' },
```

Comment:
```suggestion
      { icon: 'twitter', link: 'https://twitter.com/terramate-io' },
```

i can also change that in a follow up PR :)